README.md

```diff
@@ -150,12 +150,12 @@ cargo run --bin ln_gateway --release cfg
 ```
 To get an invoice you can use ln2 :
 ```shell
-#make sure your client holds coins for that amount tier. 100000msat = 100sat. For now the client is not able to spend with change
-lightning-cli --network regtest --lightning-dir=ln2 invoice 100000 test test 1m
+#Make sure you have pegged in small enough tiers the invoice here is 100sat and the fee will be one sat so you'll need min. 100x100000msat and 1x1000msat coins
```

Comment:
"need min. 100x100000msat and 1x1000msat coins"
not a good choice of wording, is rather confusing since it's possible with other tier/coin combinations too

Reply:
1. It's not `100x100000` but `1x100000` imo. 
2. 
 
> Make sure you have pegged in small enough tiers/denominations [whatever makes it consistent]

could be changed to
> Make sure your peg-in created coins of small enough tiers …

Reply:
1. yes right mb
2. ok